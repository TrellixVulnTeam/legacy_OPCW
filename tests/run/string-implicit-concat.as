extern def puts(str);
def main() {
    # Print out some randomly concatenated junk.
    puts("Hello "'World');
    puts('"' "Don't do that" '," said Mai.');
}
