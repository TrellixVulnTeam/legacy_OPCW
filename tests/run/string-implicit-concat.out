Hello World
"Don't do that," said Mai.
